### DISCUSSION TITLE
The Best Engine of 2022

### DISCUSSION THREAD

**Discussion by MemTheBun** [Leather128/LeathersFunkinEngine#120] [General]
More keys
Stage Editor!
Multiple Characters sibnging song1!??!?1?1?1

what a cool engine

**Comment by Leather128** [comment]
yea i basically try to add any feature I (or in a lot of cases, the people using the engine) want

**Reply by MemTheBun** [reply]
ok


Bro where is the stage editor

**Reply by dimuzicle** [reply]
download build in actions
